src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+$color-white: #fff;
+$color-hover-text: #34568b;
+$font-family: roboto, helvetica, sans-serif;
```

Comment:
There is something wrong with your default font. In your deployed version you are not using the Roboto. Maybe that's why some of your tests fail. Try to change it ;)

Conclusion: Make roboto a default font